index.html

```diff
@@ -1,11 +1,15 @@
 <!DOCTYPE html>
-<html lang="en"></html>
+<html lang="en">
+
+</html>
```

Comment:
You need  to remove this closing tag, the closing tag for **html** should be at the end of the file not here.